release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1205,6 +1203,10 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * {op-system} real time (RT) kernels are currently only supported on compute nodes, not control plane nodes. Compact clusters are not supported with RT kernels in {product-title} 4.7. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1887007[*BZ#1887007*])
 
+* It is currently not possible to use the technology preview AWS Secure Token Service (STS) in a cluster installed into the AWS C2S Secret Region due to current {product-title} limitations. This will be fixed in a future release of {product-title}. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1927157[*BZ#1927157*])
```

Comment:
use the AWS Secure Token Service (STS), which is a Technology Preview feature, in a cluster